### ISSUE TITLE
tracking_type

### ISSUE THREAD

**Issue by alle-ira** [ivoa-std/ObsCoreExtensionForRadioData#57]
Sect. 4.5 -
The parameter tracking_type as defined in ObsLocTAP pertains more to observation planning and execution that to data discovery. Additionally, its scope is wider than the radio extension since it pertains data irrespectively of their energy/frequency domain. Moreover, as it is defined now tracking_type does not comprise all the values needed to describe the variety of tracking modes associated to the sky scan modes (for instance: skydip, on the fly/raster maps).

**Comment by BaptisteCecconi** [comment]
Anyhow, this kind of list of terms should go into a vocabulary in Semantics... especially since it is used in 2 recommendations (ObsLocTAP and this one).

**Comment by Bonnarel** [comment]
I agree with Baptiste here. the parameter name should be the same whatever the situation : observation planning or description of achieved observations. this is good for interoperability. And if terms are missing they should be added in the external generic vocabulary

**Comment by alle-ira** [comment]
Apologies, what we intended is that we are not sure  this parameter should be listed in the ObsCore radio extension given its (more) general scope. Is there a use case motivating its inclusion in the radio extension?

**Comment by loumir** [comment]
Agreed with Alexandra . 
Will users query an archive using this term? 
Is this term tracking_type really designating the same concept between ObsLocTap and the tracking into a radio instrument ? 
Expanding, and standardizing  vocabularies takes time to build consensus , so is it really a key parameter ?

**Comment by mservillat** [comment]
Note that **tracking_type** is indeed used in different contexts, including our current High Energy discussions. Cherenkov telescope arrays could use this concept, as well as other ways to scan the sky (we discuss a convergent or divergent mode for example) or run observations sequences (where wobbling between 2 or 4 positions around a source in one special technique). So the High Energy domain could benefit from those descriptions for discovery purposes as well.
